lib/ObjWriter/CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ endif()
 add_library(objwriter
    SHARED
    objwriter.cpp
+   objwriter.h
```

Comment:
It doesn't work. It looks like it is a common problem for VS, [stack overflow](http://stackoverflow.com/questions/1167154/listing-header-files-in-visual-studio-c-project-generated-by-cmake)